workflow.rst

```diff
@@ -337,7 +337,8 @@ order:
 .. note::
 
     The leaving and entering events are triggered even for transitions that stay
-    in same place.
+    in same place. In addition to that, the following entering events are triggered upon the
+    workflow creation: `workflow.entered` and `workflow.[worflow name].entered`.
```

Comment:
Workflow creation is weird 

Why not : Following events are also dispatched when the subject entered the workflow for the first time

Reply:
...when the subject enters the workflow... ?

Reply:
I do not understand your answer

My first comment was in fact to remove the wording of « creation » as the workflow is not created neither create something 

That is why i proposed another wording, with the transition name « entered » and your idea of the « first » entrance :)

Reply:
Ok I get what you mean, updating.